### DISCUSSION TITLE
Share your creative HueMagic flows!

### DISCUSSION THREAD

**Discussion by Foddy** [Foddy/node-red-contrib-huemagic#254] [Show and tell]
Hi,

as some of you know, there will be a big HueMagic update in the next few days - if not the biggest one yet! I would like to generally integrate several, creative flow examples from you in the next version (and beyond) and I need your help.

You do so many cool things with HueMagic that newcomers (and me) would certainly like, inspire and help. If you think you have a cool flow to share that could help others, please share it here on this thread. Every example that makes it into the library is of course honored in some way with the username of the creator.

Whether it's "just" a cool animation in the Hue Magic node or an endless loop of chained Hue Light nodes that go on and off one after the other - everything is welcome!

**The only rule:** Please only use the nodes that are available as standard from Node-RED and HueMagic, so that everyone can run the examples directly without additional plugins.

I look forward to your flows! Many thanks for your help!

**Comment by Christoph-87** [comment]
Hi,

I like the feature to set the color temperature of the lights to automatic (depending on the time of day). What I had missed was also an automatic brightness, so that the light does not shine so bright in the middle of the night. Unfortunately, I use a time-range node, which is not in the standard and therefore does not meet the rules. Still, I thought this could be a use case for others.

**Reply by Foddy** [reply]
This feature is supported in the latest version :)
Let me know what you think and if I can optimize it a bit.

**Comment by Christoph-87** [comment]
> This feature is supported in the latest version :) Let me know what you think and if I can optimize it a bit.

Thanks a lot! That's great that you did it. I will look for a quiet minute in the next 1-2 days to update to version 4 and then test it extensively for at least 1 week and then give feedback.

**Reply by Christoph-87** [reply]
I did test it for a week now and it is exactly what I needed. My flows have been greatly simplified as a result.
